.gitignore

```diff
@@ -396,3 +396,4 @@ $RECYCLE.BIN/
 
 # setuptools-scm write_to:
 /src/pylibsshext/_scm_version.py
+.ansible
```

Comment:
Don't add arbitrary entries into the project gitignore. You can do this in your user-global gitignore if you need it but it is in no way related to the project so it shouldn't be dumped in here.
```suggestion
```